src/network/network_gui.cpp

```diff
@@ -475,6 +478,14 @@ class NetworkGameWindow : public Window {
 		this->filter_editbox.cancel_button = QueryString::ACTION_CLEAR;
 		this->SetFocusedWidget(WID_NG_FILTER);
 
+		/* As the master-server doesn't support "websocket" servers yet, we
+		 * let "os/emscripten/pre.js" hardcode a list of servers people can
+		 * join. This means the serverlist is curated for now, but it is the
+		 * best we can offer. */
```

Comment:
bit ick. Might as well disable the network window entirely, especially at the moment as there are none available at all

Reply:
I am sure some servers are going to run this to join their multiplayer, so I didn't want to disable it. But it is still a bit rough this way, I agree. Disabling it completely, I think is a bad call, and I don't really know the serverlist till it is opened.

Do you mind if we leave it like this for now, considering it more a WIP?